### ISSUE TITLE
Support for higher version of Mongoid

### ISSUE THREAD

**Issue by Bongs** [activeadmin/activeadmin-mongoid#123]
I'm trying to install activeadmin-mongoid v0.5.0 in project which use mongoid v6.4.0. It fails because of compatibility of mongoid gem version.

    Bundler could not find compatible versions for gem "mongoid":
      In Gemfile:
        mongoid (= 6.4.0)

        activeadmin-mongoid (~> 0.5.0) was resolved to 0.5.0, which depends on
          mongoid (~> 6.0.3)

**Comment by grzegorz-jakubiak** [comment]
@Bongs I don't have much time to loosen up our dependencies but you can definitely submit pull request if you want and if it passes tests and @boie0025 agrees I will merge it.

**Comment by boie0025** [comment]
I've actually tried this project with AA v6.x, and there's a bunch of failures, it'll be a rather large undertaking to get it to work I fear.  Feel free to open a PR if you do get it to work.  The projects I'm using with AA and Mongoid are all still on Mongoid v5.0 because of this.

**Comment by boie0025** [comment]
Well, I went and bumped the gem's dependency version to `~> 6.4.0`, and the specs all passed for the gem.  I think I am remembering issues that were found in specific applications using this gem (deeper testing than the smoke tests in this gem), so I think we can release a new version with the dependency set to `~> 6.0`, I'll push a branch and @Bongs and @grzegorz-jakubiak -- have a look.

**Comment by Bongs** [comment]
Thanks for the update @boie0025.

I had to comment out few things to make it work locally, this I did before your work. Somehow I managed to get it started. Those changes are not tested at all. Here is the diff -

https://github.com/activeadmin/activeadmin-mongoid/compare/master...Bongs:remove_mongoid_version_gemspec

Also not sure why it created migration file (for comments) while installing active_admin.
